src/odemis/util/test/transform_test.py

```diff
@@ -0,0 +1,636 @@
+# -*- coding: utf-8 -*-
+'''
+Created on 29 Nov 2013
+
+@author: Kimon Tsitsikas
```

Comment:
This date and copyright cannot be. Please update the date, and put Andries' name (and yours, if you've modified the code).